ensime_shared/client.py

```diff
@@ -421,6 +429,15 @@ def doc_uri(self, args, range=None):
         self.log.debug('doc_uri: in')
         self.send_at_position("DocUri", False, "point")
 
+    def usages(self, args, range=None):
```

Comment:
Ditto unrelated changes changes.

I'd prefer to align to the current pattern and change them all later on a tech-debt PR